Makefile

```diff
@@ -555,7 +557,7 @@ define collect_logs
             ./logs/* ./dynamic $${commit_info} \
             $$(find ./builds -name 'config.[hlms]*' \
                     -print); \
-        mv -f $(AT_WD)/collected_logs.tar.gz $(AT_BASE)/collected_logs-$(AT_VER_REV_INTERNAL).$(BUILD_ID)_$$(date --rfc-3339=seconds | tr ' ' '_' | tr ':' '.' | cut -c -19).tar.gz; \
+        mv -f $(AT_WD)/collected_logs.tar.gz $(AT_BASE)/$${fname}; \
```

Comment:
How about date -u +"%Y-%m-%d_%H.%M.%S" instead of using tr and cut?

Reply:
@RajalakshmiS Changed!